[enterprise-4.11] OSDOCS-3756: RN entry for OSDOCS-3621 (#46900)
openshift/openshift-docs: release_notes/ocp-4-11-release-notes.adoc

For more information, see link:https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy[Service Internal Traffic Policy].

[id="ocp-4-11-routes-spec-subdomain-field"]
==== Enhancement to the Route API

> Previously, there was no way to specify the subdomain of a Route, and the `spec.host` field was required to set the host name. Now, you can specify the `spec.subdomain` field and omit the `spec.host` field of a Route. The router deployment that exposes the Route will use the `spec.subdomain` value to determine the host name.

Reviewer: ```suggestion
Previously, you could not specify the subdomain of a route, and the `spec.host` field was required to set the host name. You can now specify the `spec.subdomain` field and omit the `spec.host` field of a route. The router deployment that exposes the route will use the `spec.subdomain` value to determine the host name.
```
